### ISSUE TITLE
"notebook.focusedRowBorder" does not work

### ISSUE THREAD

**Issue by JacksonKearl** [microsoft/vscode#125461]
Set to black, still blue.

![image](https://user-images.githubusercontent.com/8586769/120727826-6001d680-c490-11eb-8b95-97e2a4ecdc19.png)

**Comment by JacksonKearl** [comment]
Oh, maybe it's "focusedEditorBorder"? Again feel free to close if this is expected

**Comment by JacksonKearl** [comment]
More questions: `notebook.focusedRowBorder` also seems to not apply to anything

**Comment by JacksonKearl** [comment]
`"notebook.focusedRowBorder": "#00000000",` based on the description, I'd expect this to remove the top/bottom bars I see for the currently selected cell. In fact, it doesn't seem to do anything.

**Comment by rebornix** [comment]
To archive that you can do

```
"notebook.focusedCellBorder": "#fff",
"notebook.inactiveFocusedCellBorder": "#fff",
```

**Comment by misolori** [comment]
yea, so we have three different tokens for border:

- `notebook.focusedCellBorder` this is when the cell is focused
- `notebook.inactiveFocusedCellBorder` this is when the cell is NOT focused
- `notebook.focusedEditorBorder` this is for the editor when it is focused (inside the cell)
- `notebook.cellBorderColor` this is when the editor cell is NOT focused

Agreed that we can remove `notebook.focusedRowBorder` since it appears to not do anything.

**Comment by rebornix** [comment]
https://github.com/microsoft/vscode/blob/2ed7f15346f6b77794b0df1f53ee682ea915e246/src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts#L64-L74

@roblourens I'm wondering if they are oversights since they are colors in the settings editor, maybe their name should be `settings.focusedRowBorder`

**Comment by roblourens** [comment]
Fixed with 8154afb6a2a1f41eac81dfb443f75ab103f69872
